src/App.php

```diff
@@ -312,15 +316,16 @@ protected function load(DbaDefinition $dbaDefinition, ViewDefinition $viewDefini
 
 		set_include_path(
 			get_include_path() . PATH_SEPARATOR
-			. $this->getBasePath() . DIRECTORY_SEPARATOR . 'include' . PATH_SEPARATOR
-			. $this->getBasePath() . DIRECTORY_SEPARATOR . 'library' . PATH_SEPARATOR
-			. $this->getBasePath());
+			. $this->appHelper->getBasePath() . DIRECTORY_SEPARATOR . 'include' . PATH_SEPARATOR
+			. $this->appHelper->getBasePath() . DIRECTORY_SEPARATOR . 'library' . PATH_SEPARATOR
```

Comment:
None of these folders exist anymore and autoloading is done through Composer, please remove the whole `set_include_path` call.